### DISCUSSION TITLE
Flutter App

### DISCUSSION THREAD

**Discussion by minguibernardo** [community/community#19696] [Mobile]
How to run ios emulator on windows without using a mac laptop?

**Comment by abdulwahidkahar** [comment]
> How to run ios emulator on windows without using a mac laptop?

So far, when I build a **Flutter** (Multiplatform) application on windows, I also haven't found how to run the ios simulator on windows. I think the iOS simulator can only run on MacOs.
I hope this helps

**Reply by minguibernardo** [reply]
thanks

**Comment by JhonnyLusonode** [comment]
Hello @minguibernardo you have a lot of IOS emulators for windows just as Smartface.

I belive you can connect it with vscode as an extension not sure..

Here is the link to their website: [Smartface](https://smartface.io/)

I haven´t used it yet but i´ve heard really good things about it.

Hope it helps!
If you find this answer useful please consider marking it as "Answered".
(btw, this would help me to get the badge. Thank you! 😁)
